INSTALL.txt

```diff
@@ -1,30 +1,29 @@
   Build requirements (Installing from source)
-    For building Rakudo you need at least a C compiler, a "make" utility,
-    and Perl 5.10 or newer. To automatically obtain and build MoarVM
-    as well as NQP, you may also need a git client, which is also needed
-    for fetching the test suite.
+    For building Rakudo you need at least a C compiler, a "make" utility, and
+    Perl 5.10 or newer. To automatically obtain and build MoarVM as well as NQP,
+    you may also need a git client, which is also needed for fetching the test
+    suite.
 
-    Building rakudo can take up to 1G of memory when compiling for the
-    MoarVM runtime. The requirements are higher for the JVM backend.
+    Building rakudo can take up to 1G of memory when compiling for the MoarVM
+    runtime. The requirements are higher for the JVM backend.
 
-    (Perl is installed by default already). To enable parallel testing you
-    also need the CPAN module Test::Harness in version 3.16 or newer; you
-    can control the number of parallel jobs with the "TEST_JOBS" environment
-    variable. If TEST_JOBS is not specified, 6 jobs will be used.
+    (Perl is installed by default already). To enable parallel testing you also
+    need the CPAN module Test::Harness in version 3.16 or newer; you can control
+    the number of parallel jobs with the "TEST_JOBS" environment variable. If
+    TEST_JOBS is not specified, 6 jobs will be used.
 
   Building and invoking Rakudo
-    If you're wanting the bleeding-edge version of the Rakudo Perl 6
-    compiler, we recommend downloading Rakudo directly from Github
-    and building it from there.
+    If you're wanting the bleeding-edge version of the Rakudo Perl 6 compiler,
```

Comment:
There's another _Perl 6_ lurking here.

Reply:
@patrickbkr Thanks! Took your comments into account with e1db95e. Except for the Perl version at least for now.